indico/modules/events/abstracts/templates/invited_abstract.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'events/abstracts/display/_base.html' %}
+
+{% from 'message_box.html' import message_box %}
+{% from 'forms/_form.html' import form_header, form_rows, form_footer, form_row %}
+
+{% block page_content_class %}{% endblock %}
+
+{% block title %}
+    {% trans %}
+        Submit an abstract
+    {% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {% call message_box('info') -%}
+        {% trans %}
+            You have been invited to submit an abstract in this event.
+            After you submit the abstract it will subject to the reviewing process.
+        {% endtrans %}
+    {%- endcall %}
+    {% if session.user and session.user != abstract.submitter %}
+        {% call message_box('warning') -%}
+            {% trans submitter_name=abstract.submitter.full_name %}
+                You are submitting an abstract on behalf of {{ submitter_name }}.
+                Make sure you have been approved to do that.
```

Comment:
Maybe something like "Please make sure this is really what you want to do." fits better there? This implies that you are authorized (or just going to do it anyway knowingly). The current wording could result in people asking "how do i get approved to do that" IMO

Reply:
Yeah, I wasn't sure about that as well